server/service/src/programs/patient/patient_updated.rs

```diff
@@ -38,6 +38,7 @@ pub fn create_patient_name_store_join(
 /// Updates the names table for the updated patient.
 pub fn update_patient_row(
     con: &StorageConnection,
+    store_id: Option<String>,
```

Comment:
Made it optional since this function is used in `sync_upsert_document` as well...